povm_toolbox/library/povm_implementation.py

```diff
@@ -41,15 +41,18 @@ class POVMImplementation(ABC, Generic[MetadataT]):
     def __init__(
         self,
         n_qubit: int,
+        qubit_specifier: list[int] | None = None,  # TODO: add | Layout
     ) -> None:
         """Initialize the POVMImplementation.
 
         Args:
             n_qubit: number of logical qubits in the system.
+            qubit_specifier: list of index specifying on which qubits the POVM acts.
         """
         super().__init__()
         self.n_qubit = n_qubit
         # TODO: add qubit_specifier or layout to apply, see issue #15
+        self.idx_layout = qubit_specifier
```

Comment:
If we make this a public attribute, we should find a good name. As of now, I find it confusing to see a name different than the `__init__` argument.
That said, not sure if `qubit_specifier` is ideal. I am open to suggestions.

Maybe even `layout` or `measurement_layout`?

Reply:
`measurement_layout` sounds good 👍